docs/USAGE.md

```diff
@@ -27,50 +27,50 @@
 
 JSON-Schema-faker (or `jsf` for short) combines two things:
 
- * The [JSON-schema specification](http://json-schema.org/draft-04/json-schema-core.html), that defines what is the allowed content of a JSON document
- * Fake data generators, that are used to generate basic or complex data, conforming to the schema.
+- The [JSON-schema specification](http://json-schema.org/draft-04/json-schema-core.html), that defines what is the allowed content of a JSON document
+- Fake data generators, that are used to generate basic or complex data, conforming to the schema.
```

Comment:
Some of these formatting changes are because I have MarkdownLint installed. I can go back and remove the formatting changes if that's important to you.

Reply:
This is fine, thank you!